pyproject.toml

```diff
@@ -1,25 +1,31 @@
-[tool.poetry]
+[project]
 name = "uptick-splat"
 version = "0.4.1"
 description = "Django library for invoking splat"
-authors = ["william chu <william.chu@uptickhq.com>"]
+authors = [
+    { name = "william chu", email = "william.chu@uptickhq.com" }
+]
 readme = "README.md"
-packages = [{ include = "uptick_splat" }]
-
-[tool.poetry.dependencies]
-python = "^3.9"
-django = ">=3.1, <5.0.0"
-boto3 = "*"
-playwright = "1.43.0"
-
+requires-python = ">=3.9"
+dependencies = [
+    "django>=3.1, <5.0.0",
+    "boto3",
+    "playwright==1.43.0",
+]
 
-[tool.poetry.group.dev.dependencies]
-pytest = "^8.1.1"
-ruff = "^0.3.7"
+[project.optional-dependencies]
+dev = [
```

Comment:
```
[dependency-groups]
dev = [
   "pytest>=8.1.1",
    "ruff>=0.3.7",
    "requests>=2.31.0",
]
```
^ this is probably what you want. This _should_ make `uv run ruff ...` work out of the box (`uv run` and `uv sync` include the `dev` dependency group by default)